src/h3lib/lib/h3Index.c

```diff
@@ -557,6 +557,11 @@ H3Index _faceIjkToH3(const FaceIJK* fijk, int res) {
 
     // check for res 0/base cell
     if (res == 0) {
+        if (fijk->coord.i > 2 || fijk->coord.j > 2 || fijk->coord.k > 2) {
```

Comment:
Can we have a well-named constant for `2` here?

Reply:
+1 well-named constant and maybe a comment on why > 2 for any of the IJK is bad too :D ?